src/runtime/bootstrap-custom-element.ts

```diff
@@ -57,8 +56,8 @@ export const proxyCustomElement = (Cstr: any, compactMeta: d.ComponentRuntimeMet
     if (BUILD.appendChildSlotFix) {
       patchSlotAppendChild(Cstr.prototype);
     }
-    if (BUILD.scopedSlotTextContentFix) {
-      patchTextContent(Cstr.prototype, cmpMeta);
+    if (BUILD.scopedSlotTextContentFix && cmpMeta.$flags$ & CMP_FLAGS.scopedCssEncapsulation) {
```

Comment:
This additional condition is the same one that was a top-level check in the `patchTextContent` method. Just moved it here so we don't need to change things later when we make the `experimentalSlotFixes` the default behavior and remove these other `extras` options.